translations/stripes-erm-components/de.json

```diff
@@ -20,5 +20,6 @@
     "licenseCard.status": "Status",
     "licenseCard.startDate": "Start date",
     "licenseCard.endDate": "End date",
-    "licenseCard.licensor": "Licensor"
+    "licenseCard.licensor": "Licensor",
+    "licenseTermsList.notSet": "Not set"
 }
```

Comment:
These changes are not needed.  (In fact, changes to translation files other than `en.json` will be automatically overwritten when the Lokalise-to-GitHub script is run.)  In other words, these changes are harmless, but they will not "stick".